pub_sub/go/sdk/checkout/app.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	dapr "github.com/dapr/go-sdk/client"
+)
+
+var (
+	PUBSUB_NAME  = "order_pub_sub"
+	PUBSUB_TOPIC = "orders"
+)
+
+func main() {
+	client, err := dapr.NewClient()
+	if err != nil {
+		panic(err)
+	}
+	defer client.Close()
+	ctx := context.Background()
+	for i := 1; i <= 10; i++ {
+		order := `{"orderId":` + strconv.Itoa(i) + `}`
+
+		// Publish an event using Dapr pub/sub
+		if err := client.PublishEvent(ctx, PUBSUB_NAME, PUBSUB_TOPIC, []byte(order)); err != nil {
+			panic(err)
+		}
+
+		fmt.Println("Published data: ", string(order))
```

Comment:
Isn't `order` already a string? No need to cast it.

Reply:
I kept it as string because the readme tests(using mechanical markdown) are not passing.

Reply:
I'm just saying that you don't need to cast is with `string(order)`.